app/services/podcasts/feed.rb

```diff
@@ -44,9 +44,8 @@ def set_unreachable(status: :unreachable, force_update: false)
     # If the episodes URLs are still reachable, the podcast will remain on the site.
     # If they are not, the podcast will be hidden.
     def refetch_items
-      podcast.podcast_episodes.find_each do |episode|
-        PodcastEpisodes::UpdateMediaUrlWorker.perform_async(episode.id, episode.media_url)
-      end
+      job_params = podcast.podcast_episodes.pluck(:id, :media_url)
```

Comment:
Reading note: `pluck` returns the array of arrays structure we need for `perform_bulk`.